tests/runtime/builtin

```diff
@@ -96,18 +96,32 @@ EXPECT SUCCESS .*
 TIMEOUT 5
 AFTER ./testprogs/syscall read
 
-NAME func
-PROG k:vfs_read { printf("SUCCESS %s\n", func); exit(); }
-EXPECT SUCCESS .*
+NAME func_kprobe
+PROG k:vfs_read { printf("func: \"%s\"\n", func); exit(); }
+EXPECT func: "vfs_read"
+TIMEOUT 5
+AFTER ./testprogs/syscall read
+
+NAME func_kretprobe
```

Comment:
Shouldn't this test have a requirement for the `get_func_ip` helper?

Reply:
Yeah if you add a requirement on the feature then we don't need to disable it. It'll only run if the host supports the correct version.